### PR TITLE
Add to correct with sbom-info.yaml

### PR DESCRIPTION
**Pull request by dd-jy**
## Description

Add to correct with sbom-info.yaml

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/fosslight_source/_help.py [hunk 1]
```diff
@@ -28,7 +28,9 @@
             -s <scanner>\t   Select which scanner to be run (scancode, scanoss, all)
             -j\t\t\t   Generate raw result of scanners in json format
             -t <float>\t\t   Stop scancode scanning if scanning takes longer than a timeout in seconds.
-            -c <core>\t\t   Select the number of cores to be scanned with ScanCode."""
+            -c <core>\t\t   Select the number of cores to be scanned with ScanCode.
+            --no_correction\t   Enter if you don't want to correct OSS information with sbom-info.yaml
+            --correct_fpath\t   Path to the sbom-info.yaml file"""
```
Comment (soimkim) [inline src/fosslight_source/_help.py:33]: @dd-jy , --correct_fpath <path>\t 로 변경 필요해보입니다.

Comment (dd-jy) [reply src/fosslight_source/_help.py:33]: @soimkim 
변경 필요한 사유가 있을까요? help 메시지 정렬과 동작에 이상 없어보여 문의드립니다.

Comment (soimkim) [reply src/fosslight_source/_help.py:33]: `<path> `를 입력하면 깃헙에서 뭔가 tag인지 안보이네요

Comment (dd-jy) [reply src/fosslight_source/_help.py:33]: https://github.com/fosslight/fosslight_source_scanner/pull/122 에 수정하였습니다~

**Pull request merged**